widget_bandsplot/schemas/pdos.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$id": "https://raw.githubusercontent.com/osscar-org/widget-bandsplot/fea/xx/add-schema/widget_bandsplot/schemas/pdos.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "JSON Schema for a PDOS entry",
+    "type": "object",
+    "required": ["dos"],
+    "properties": {
+        "fermi_energy": {"type": "number"},
+        "dos": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "required": ["label", "x", "y"],
+                "properties": {
+                    "label": {"type": "string"},
+                    "x": {
+                        "type": "array",
+                        "items": {"type": "number"}
+                    },
+                    "y": {
+                        "type": "array",
+                        "items": {"type": "number"}
+                    },
+                    "lineStyle": {"enum": ["solid", "dash"]},
+                    "borderColor": {"type": "string"},
```

Comment:
It is possible to verify whether it is a valid color string?

Reply:
This depends on which values are allowed, I think currently in the javascript, the hex is used for setting the color.
If this is the case, we can have the schema for RGB color as: https://github.com/the-grid/apidocs/issues/26 

```json
"hexcolor": {
  "description": "RGB color",
  "format": "rgbcolor",
  "pattern": "^#([a-fA-F0-9]{6}|[a-fA-F0-9]{3})$",
  "example": "#aa33cc",
  "type": "string"
},
```

Reply:
I was asking what values are supported by the library you used in JavaScript, how many colors name are supported there?

Reply:
We are going to support only common colors like red, blue, black, green or more such as list here https://matplotlib.org/stable/gallery/color/named_colors.html?

Reply:
"When supplying colors to Chart options, you can use a number of formats. You can specify the color as a string in hexadecimal, RGB, or HSL notations. "

Reply:
https://www.w3schools.com/colors/colors_names.asp

Reply:
I see, then I'd say let's keep the string without any pattern, it's not easy to support all of those explicitly.